docs/source/development/creating_an_example_notebook.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Creating an Example Notebook\n",
     "\n",
     "Functionality of ProbNum is explained in detail in the form of Jupyter notebooks in the\n",
-    "[tutorials section](../development/developer_guides.rst) of the documentation.\n",
+    "[tutorials section](../development/tutorials.rst) of the documentation.\n",
```

Comment:
there is no link to the "Tutorials and Examples" section of the documentation, since it does not have its own overview page. This should therefore not have a link (in particular not to the development guides). I would also suggest not renaming `development/developer_guides.rst` to `tutorials.rst` since these are not called tutorials but developer guides currently.

Reply:
makes sense, I'll change it back.

Reply:
I guess I am just not a super big fan of using the word "developer" or "development" as it can sound quite daunting for beginners. "Contributing", or tutorials for contributing etc as sound a bit more welcoming.

Reply:
but it's not a bit point :)